database/src/migrations/release.0.23/populate_submission_message_type.sql

```diff
@@ -11,7 +11,6 @@ insert into submission_message_type (name, record_effective_date, description, s
 
 -- field related
 insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Missing Required Field', now(), 'A required field was not detected.', (select submission_message_class_id from submission_message_class where name = 'Error'));
-insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Unexpected Format', now(), 'A field with an unexpected format was detected.', (select submission_message_class_id from submission_message_class where name = 'Error'));
+insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Unexpected Format', now(), 'A field with an unexpected format was detected.', (select submission_message_class_id from submission_message_class where name = 'Warning'));

```

Comment:
Why the change to warning?

I would assume a regex rule would be an error, otherwise why bother with it at all.  We could just leave it at being a string or something

Reply:
we need to show something for the warning category ... should I duplicate is with a different name?